### PR TITLE
Add about Sourcegraph section to ABC book landing page

### PR DESCRIPTION
**Pull request by aileenrose**
Screen Shot 2020-06-02 at 3 17 36 PM

### CODE REVIEW

File: website/src/pages/resources/abcs-book.tsx [hunk 1]
```diff
@@ -111,6 +111,26 @@ export default ((props: any) => (
                     </p>
                 </div>
             </div>
+            <hr />
+            <div className="row justify-content-md-center mt-5">
+                <div className="col-md-10">
+                    <h3 className="text-center">About Sourcegraph</h3>
+                    <p>
+                        Sourcegraph empowers all developers with{' '}
```
Comment (adaaaam) [inline website/src/pages/resources/abcs-book.tsx:119]:
```suggestion
                        Sourcegraph empowers all developers to explore, navigate and better understand all code, faster, with{' '}
```

File: website/src/pages/resources/abcs-book.tsx [hunk 2]
```diff
@@ -111,6 +111,26 @@ export default ((props: any) => (
...
+                        <a href="/universal-code-search">universal code search</a>, intelligence, and automation. Code
```
Comment (adaaaam) [inline website/src/pages/resources/abcs-book.tsx:120]:
```suggestion
                        <a href="/universal-code-search">Universal Code Search</a>. 
```

File: website/src/pages/resources/abcs-book.tsx [hunk 3]
```diff
@@ -111,6 +111,26 @@ export default ((props: any) => (
...
+                        intelligence can help bring the future sooner, and Sourcegraph’s mission is to provide tools
```
Comment (adaaaam) [inline website/src/pages/resources/abcs-book.tsx:121]:
```suggestion
                        Sourcegraph’s mission is to make it easier and faster for developers to work on solving problems.
```

File: website/src/pages/resources/abcs-book.tsx [hunk 4]
```diff
@@ -111,6 +111,26 @@ export default ((props: any) => (
...
+                        that let developers from any organization create quality products by leveraging the best
```
Comment (adaaaam) [inline website/src/pages/resources/abcs-book.tsx:122]:
```suggestion
                        
```

File: website/src/pages/resources/abcs-book.tsx [hunk 5]
```diff
@@ -111,6 +111,26 @@ export default ((props: any) => (
...
+                        technology available.
```
Comment (adaaaam) [inline website/src/pages/resources/abcs-book.tsx:123]:
```suggestion
                      
```

**Comment by aileenrose** [comment]
looks great:
https://deploy-preview-1021--sourcegraph.netlify.app/resources/abcs-book/

**Pull request merged**